pages/item/_itemId/index.vue

```diff
@@ -144,6 +145,7 @@
                       single-line
                       hide-details
                     >
+                      <!-- eslint-disable-next-line vue/no-template-shadow -->
```

Comment:
There must be a better way to fix this

Reply:
@Camc314 I changed the variable's name and it's references and It doesn't worked, everything was undefined. I'm not really sure what ``slot-proposition`` does at all, maybe @mrtimscampi can shed some light on this.

Reply:
You can rename variable while destructuring.

`slot-scope` defines variables that are passed to the scope. So here, we pass `items` to the `v-select`, which then iterates through them. Its template has a slot, into which is passe the item. That's what you're getting in `slot-scope`.

You can do this to rename the variable:
```vue
<template slot="selection" slot-scope="{ item: mediaSourceItem }">
```